packages/manager/apps/zimbra/src/api/POST/apiv2/services.ts

```diff
@@ -0,0 +1,13 @@
+import { apiClient } from '@ovh-ux/manager-core-api';
+import { OrganizationBodyParams } from '@/api/api.type';
+
+export const postZimbraPlatformOrganization = async (
+  platformId: string,
+  params: OrganizationBodyParams,
+) => {
+  const { data } = await apiClient.v2.post<void>(
```

Comment:
I had to take an example from another project. But in fact I have the impression that for post / put / delete typing isn't essential because we have an axios type object which is returned to us.